### PR TITLE
make graphql typing for getDescription conform with graphql-js

### PR DESCRIPTION
**Pull request by divyenduz**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/graphql/graphql-js/blob/master/src/utilities/buildASTSchema.js#L465, https://github.com/graphql/graphql-js/pull/1165
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by alessiodionisi — CHANGES_REQUESTED** [review]
My advice is to preserve the original structure so it would be useful to create and export type `GraphQLSchemaValidationOptions` in `../type/schema.d.ts` (like https://github.com/graphql/graphql-js/blob/master/src/type/schema.js#L233) and extend in `BuildSchemaOptions` (like https://github.com/graphql/graphql-js/blob/master/src/utilities/buildASTSchema.js#L76).

File: types/graphql/utilities/buildASTSchema.d.ts [hunk 1]
```diff
@@ -18,7 +24,10 @@ export function buildASTSchema(ast: DocumentNode): GraphQLSchema;
  * Given an ast node, returns its string description based on a contiguous
  * block full-line of comments preceding it.
  */
-export function getDescription(node: { loc?: Location }): string;
+export function getDescription(
+    node: { description?: StringValueNode; loc?: Location },
+    options: BuildSchemaOptions
+  ): string;
```
Comment (IvanGoncharov) [inline types/graphql/utilities/buildASTSchema.d.ts]: Please use two spaces as indentation to be consistent with `graphql-js`.

File: types/graphql/utilities/buildASTSchema.d.ts [hunk 2]
```diff
@@ -18,7 +24,10 @@ export function buildASTSchema(ast: DocumentNode): GraphQLSchema;
  * Given an ast node, returns its string description based on a contiguous
  * block full-line of comments preceding it.
  */
```
Comment (IvanGoncharov) [inline types/graphql/utilities/buildASTSchema.d.ts]: Also updated comment to keep it in sync with graphql-js:
https://github.com/graphql/graphql-js/blob/master/src/utilities/buildASTSchema.js#L456-L464

**Comment by divyenduz** [comment]
Thanks for the feedback @adnsio, @IvanGoncharov! 

I have made the requested changes. Please let me know if any further changes are needed.

**Review by IvanGoncharov — APPROVED** [review]

**Review by alessiodionisi — APPROVED** [review]

**Pull request merged**